studio/components/interfaces/Settings/General/CustomDomainConfig/CustomDomainConfig.tsx

```diff
@@ -49,7 +52,11 @@ const CustomDomainConfig = () => {
                 icon={<IconAlertCircle size={18} strokeWidth={1.5} />}
                 primaryText="Custom domains are a Pro plan add-on"
                 projectRef={ref as string}
-                secondaryText="To configure a custom domain for your project, please upgrade to a Pro plan"
+                secondaryText={
+                  tier === PRICING_TIER_PRODUCT_IDS.FREE
+                    ? 'To configure a custom domain for your project, please upgrade to the Pro plan and purchase the add-on'
```

Comment:
"To configure a custom domain for your project, please upgrade to the Pro plan _with the custom domains add-on selected_"

You purchase the add-on in the same step as upgrading to pro, right? Currently, it sounds like two different steps.
Feel free to change my suggestion if you think there's an even clearer way of explaining it :)

Reply:
oh yup! that does sound better, thanks man will update